### ISSUE TITLE
Search in the playback tab seems to be broken

### ISSUE THREAD

**Issue by Hajuskin** [jcorporation/myMPD#681]
myMPD version: 9.1.1

**Describe the bug**

The search in the playback tab doesn't return all valid results, please have a look at the screenshots:
![2022-02-12_213738](https://user-images.githubusercontent.com/51139914/153727824-01ba1b45-68bd-4396-92e4-009af11e5f9e.png)
![2022-02-12_213745](https://user-images.githubusercontent.com/51139914/153727828-8c3f3c4e-c6e7-48f0-8d98-14a460d580c9.png)
It doesn't seems to make a difference if you search for `Any Tag` or `Title` etc.
The shown number of search results is also off by one (see first screenshot)

Entering more than one letter when searching for a `Filename` results in this error:
![2022-02-12_215000](https://user-images.githubusercontent.com/51139914/153728217-377288c5-8e0a-4a16-901d-d84e6600e09a.png)

Log for filename search shows `WARN     mympdapi  Unknown tag filename`
```
2022-02-12T21:05:40.156199065Z INFO     webserver HTTP request (18): POST /api/
2022-02-12T21:05:40.156296601Z DEBUG    webserver API request (18): {"jsonrpc":"2.0","id":0,"method":"MYMPD_API_QUEUE_SEARCH","params":{"filter":"filename","offset":0,"limit":500,"searchstr":"ab","cols":["Pos","Title","Artist","Album","Duration","Date","Genre"]}}
2022-02-12T21:05:40.156318768Z INFO     webserver API request (18): MYMPD_API_QUEUE_SEARCH
2022-02-12T21:05:40.168935359Z DEBUG    mympdapi  Leaving mpd idle mode
2022-02-12T21:05:40.169127136Z DEBUG    mympdapi  Handle request
2022-02-12T21:05:40.169159321Z INFO     mympdapi  MYMPD API request (18)(0) MYMPD_API_QUEUE_SEARCH: {"jsonrpc":"2.0","id":0,"method":"MYMPD_API_QUEUE_SEARCH","params":{"filter":"filename","offset":0,"limit":500,"searchstr":"ab","cols":["Pos","Title","Artist","Album","Duration","Date","Genre"]}}
2022-02-12T21:05:40.169181025Z WARN     mympdapi  Unknown tag filename
2022-02-12T21:05:40.169204136Z WARN     mympdapi  Validation of value for JSON path "$.params.filter" has failed
2022-02-12T21:05:40.169228839Z ERROR    mympdapi  Error processing method "MYMPD_API_QUEUE_SEARCH"
2022-02-12T21:05:40.169245487Z DEBUG    mympdapi  Push response to web_server_queue for connection 18: {"jsonrpc":"2.0","id":0,"error":{"method":"MYMPD_API_QUEUE_SEARCH","facility":"general","severity":"error","message":"Validation of value for JSON path \"$.params.filter\" has failed","data":{}}}
2022-02-12T21:05:40.169264265Z DEBUG    mympdapi  Entering mpd idle mode
2022-02-12T21:05:40.206512320Z DEBUG    webserver Got API response for id "18"
2022-02-12T21:05:40.206557412Z DEBUG    webserver Sending response to conn_id 18 (length: 195): {"jsonrpc":"2.0","id":0,"error":{"method":"MYMPD_API_QUEUE_SEARCH","facility":"general","severity":"error","message":"Validation of value for JSON path \"$.params.filter\" has failed","data":{}}}
2022-02-12T21:05:40.206626319Z DEBUG    webserver Sending 195 bytes to 18
```

**Comment by jcorporation** [comment]
You found 2 bugs:
- filename was not allowed as tag
- the pagination of the queue search results omits the first entry

Both are fixed in devel and upcoming v9.2.0 branch.

Can you check if this fixes all your issues?

**Comment by Hajuskin** [comment]
Looks good to me!
